### DISCUSSION TITLE
奇忽局09号理事会：声音实践理事会

### DISCUSSION THREAD

**Discussion by Tz-H** [qihuju/qihuju.github.io#34] [General]
【发起人】: @Tz-H 

【发起缘由】：

> 讨论声音/音乐的这一期是否也可以介入讨论 ASMR 这个文化现象载体？
> https://en.wikipedia.org/wiki/ASMR
>
> 我一直非常好奇听觉是如何对人的意识和情绪产生影响的，所谓“好听”/“难听”的根源又是什么，有机会的话，想和大家一起讨论研究。
>
> 那我们就攒出个关于声音的专题局出来呗~
> 以 ASMR 还有音乐 DJ 的音乐段子，经历/经验分享作为主题，辅以声音/音乐/文化/意识这个层面的后续讨论

【话题与回复】：

1. 声音与心流的关系
   > 我总感觉这个和“心流”有关系，它就像一段程序/病毒，hack黑入了人的心流，从而影响改造了人的情绪和认知 —— @Tz-H 
   > 感觉就是人类肉体的bug —— @思洋

2. [ASMR](https://en.wikipedia.org/wiki/ASMR)
   > asmr和性欲联系的蛮紧的 —— @Coraline
   > 听过段时间ASMR。记得有说法是大脑对近距离声音的危险判定，产生颅内高潮 —— @思洋

3. 这种所谓的“好听”与“难听”（可以类比视觉上的“好看”与“难看”），究竟是生物学\遗传学意义上的先天印记，还是社会学意义上的后天归训文化习得，亦或是二者兼有之？
   > 这个“好”是一个很主观的判断，但科学界会认为说，如果是主观的，也就是内在的，那么即可通过外在方式来测量这种主观判断 —— @mengtaizhang 
   > dj有很多“规则” 比如哪个调混哪个调的曲会“悦耳”  哪个bpm适合热场哪个bpm适合高潮 其实也就是总结的人普遍觉得“好听能跳”的规律  但我认识的dj都会刻意打破其中的一个或多个“规则”并说自己并不是不懂这个，只是dont give a shit —— @MingxuanXie 

4. 建立在听觉基础之上的文化元素，是否也有类似视觉传播上的模因 Meme 存在呢？如果有，它的“生存”/“传播”范式是怎样的？
   > 某种程度上dj就是在玩声音meme —— @MingxuanXie 
   >
   > 要是Clubhouse聊这个话题的时候，房间能请一个DJ来，现场播一段解释说明一段，播一段解释说明一段，一定会很有趣 —— @Tz-H 
   > 不过我们的本意不是做一个musician房，做clubhouse舞池，而是请DJ来讲解利用音乐的“段子”与舞池交流的这种操作行为背后的机制和观察效果，这个感觉会很有意思。。。—— @Tz-H 
   >
   > 當代的作曲思維剛好跟DJ相反，我們會預測聽眾在某個橋段想聽什麼，就故意就不演出來，或把某個很迷你的元素拉得很長讓人有種坐立難安的心理，也會造成另類的高潮 —— @YanYangz 
   >
   >  学校教音乐艺术的老师分享的多年前实验性音乐 [Maryanne Amacher ‎– Sound Characters (Making The Third Ear) - 1999](https://vimeo.com/88559973)(好像中间有通过高频波段？创造幻觉)，不知道有没有关于这方面的探讨 —— @Yiru-SHA 

5. [《哥德尔、埃舍尔、巴赫——集异壁之大成》](https://book.douban.com/subject/1291204//)
   > [哥德尔、埃舍尔、巴赫 | 混乱博物馆](https://mp.weixin.qq.com/s/dLmNYeGOl8qjXwycEALThg) —— @Lupingyue 

【成员列表】：

- @Tz-H 
- @Lupingyue 
- @DocRace 
- @MingxuanXie 
- @YanYangz 
- @Yiru-SHA 
- @mengtaizhang 
- @Sleeper2060

**Comment by Lupingyue** [comment]
抢到沙发🛋️✌️

@Sleeper2060 ，看 @Tz-H 列的名单，目前你的帮派已经事第二了😢

**Comment by DocRace** [comment]
期待一下

**Reply by Yiru-SHA** [reply]
提前撒花

**Reply by DocRace** [reply]
我们自己做过chill和带有一点点asmr助眠的产品

**Comment by YanYangz** [comment]
簽到🙋🏻‍♀️

**Comment by Sleeper2060** [comment]
John Cage的作品4'33"，一部没有“声音”的声音艺术作品。
他和杜尚有一件很出名的作品“Reunion,”用下象棋来做类似声音collage的作品，看似随机的声音来自有策略的下棋战术。也算是达达主义作品。
很多时候觉得声音要有时间这个载体，没有时间它连collage都算不上。

**Reply by Lupingyue** [reply]
John Cage和杜尚的思维和实践已经很接近现在科技哲学的讨论方向了，这两件作品都很有启发，特别是Reunion

**Comment by MingxuanXie** [comment]
声音方面个人感兴趣 live coding / Algorave 这种实时编程形式也是把创作时间接入，某种程度上把编程者作为一个实时参数、变量而不是一个commader控制着音乐

live coding / algorave 之前在英国火了一段时间 
卫报的介绍 https://www.youtube.com/watch?v=h340aNznHnM&ab_channel=GuardianCulture
algorave先锋Alex McLean这个视频讲到一些对编程很有趣的看法（不是用编程解决问题而是用编程创造问题） https://www.youtube.com/watch?v=nAGjTYa95HM&ab_channel=TEDxTalks

我觉得未来的赛博建筑师也会是live coder——不是事先设计好空间，而是根据现场人的反应实时编码出虚拟空间

**Reply by Lupingyue** [reply]
> 
> 我觉得未来的赛博建筑师也会是live coder——不是事先设计好空间，而是根据现场人的反应实时编码出虚拟空间

很喜欢这一句👍

**Reply by Sleeper2060** [reply]
有一部好像是德国电影，叫异界/koma，就是一个建筑师可以打破物理定律，实时新建任何的空间建筑。电影还ok。

**Reply by Lupingyue** [reply]
> 有一部好像是德国电影，叫异界/koma，就是一个建筑师可以打破物理定律，实时新建任何的空间建筑。电影还ok。

必须重新定义“建筑师”了，昨天 @MingxuanXie 在房间里引用了一句话很震撼：一半的建筑师都应该停止设计工作，转而设计iPhone app

**Comment by Tz-H** [comment]
声音实践项目01：奇忽局 QIHUJU | Mixcloud 

[https://www.mixcloud.com/qihuju/](https://www.mixcloud.com/qihuju/)

上传并保存了奇忽局Clubhouse录音房的音频节目
